manifests/rules/out/all.pp

```diff
@@ -0,0 +1,7 @@
+# allow all outbound
+class nftables::rules::out::all {
+  nftables::rule{
+    'default_out-all':
+      content => 'accept',
```

Comment:
this should probably have a high order like 90, so one can still easily put deny statements in front.

Reply:
Yes makes sense, will change when I take action on first one above.